.github/workflows/build.yml

```diff
@@ -60,12 +60,27 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Install macOS dependencies
-        if: ${{ matrix.os.preset == 'mac' }}
+      - name: Install macOS Rosetta 2
+        if: ${{ matrix.os.name == 'macOS-Intel' }}
+        run: /usr/sbin/softwareupdate --install-rosetta --agree-to-license
+
+      - name: Install Intel version of Brew
+        if: ${{ matrix.os.name == 'macOS-Intel' }}
+        shell: arch -x86_64 /bin/bash -e {0}
+        run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+      - name: Install macOS Intel dependencies
+        if: ${{ matrix.os.name == 'macOS-Intel' }}
+        shell: arch -x86_64 /bin/bash -e {0}
         run: |
-          # Install packages from Homebrew
+          eval "$(/usr/local/bin/brew shellenv)"  # makes the Intel version of Brew and its installs are enshrined as the default 
+          echo "PATH=$PATH" >> $GITHUB_ENV  # modify $PATH so the above step persists into the future steps
           brew bundle install
 
+      - name: Install macOS ARM dependencies
+        if: ${{ matrix.os.name == 'macOS-ARM' }}
```

Comment:
we need to run `brew bundle install` for both builds, I'd suggest removing this condition as well as l.78 that does the same thing for intel

Reply:
@Lgt2x, fixed and ready for re-review!

I thought that I had to also run `brew bundle install` with the specific x86_64 shell when compiling for Intel, but apparently all that is necessary is to install the Intel version of Brew and use it.